### ISSUE TITLE
Multiple Queues BadSignature exception

### ISSUE THREAD

**Issue by masb3** [django-q2/django-q2#244]
I have several instances (let's say 2) serving single django app and use single MongoDB broker.  
On 1st instance settings 
```
Q_CLUSTER = {"name": "queue1", .....}
``` 
and 2nd  
```
Q_CLUSTER = {"name": "queue2", .....}
```  
On the 1st instance I call `async_task("my_task", cluster="queue2")` , but then on 2nd instance I got error:  
```
[Q] ERROR Failed to push task to queue
...
django.core.signing.BadSignature: Signature "xxxx" does not match
```
Seems it works if i configure `ALT_CLUSTERS: {"queue2": ...}` on 1st instance.  
Will the 1st instance also serve tasks from "queue2"?  
Should I also config ALT_CLUSTERS on  all instances listing the queues on other instances?

**Comment by GDay** [comment]
Yeah, the name is used for signing, so this is expected behavior. You should indeed use the `ALT_CLUSTERS` for this. 

>  Will the 1st instance also serve tasks from "queue2"?

No, if you define ALT_CLUSTERS and point the cluster to one queue, then it will not run jobs from the other queue.

You can deploy multiple `qclusters` instances with the same name. Those will all run jobs from the same queue.

**Comment by masb3** [comment]
@GDay thanks for explanations.

**Comment by masb3** [comment]
@GDay I would ask one more question. Looking into the code it is possible to define `cluster_name`
```
Q_CLUSTER = {'name': 'common name', 'cluster_name': 'queue1'}
```
taking into account I would run multiple clusters on different instances, I can do the following on another instance:
```
Q_CLUSTER = {'name': 'common name', 'cluster_name': 'queue2'}
```
That configuration allows me to push tasks from instance 1 to 2 and vice versa, since the `name` is the same. Are there any drawback comparing to use of `ALT_CLUSTERS` (that will require me to refactor start.sh scripts to start `qcluster` with different `--name queue6` args)?

**Comment by GDay** [comment]
Not sure I am following. If you run multiple clusters with different queue names, then you need `cluster_name`.

**Comment by masb3** [comment]
@GDay yeah I think I got it.
